ibm/mas_devops/roles/cluster_monitoring/README.md

```diff
@@ -13,7 +13,7 @@ The credentials for the grafana admin user are stored in `grafana-admin-credenti
 Role Variables
 -------------------------------------------------------------------------------
 ### cluster_monitoring_action
-Inform the role whether to perform an install or an uninstall of cluster monitoring.
+Inform the role whether to perform an install or an uninstall of cluster monitoring. Can also be set to update_grafana to update the Grafana Operator from V4 to V5.
```

Comment:
Does the update keep the login data that might be setup? I don't think it would as the PVC would be new and in a different namespace. So if uses have setup users in v4 then updating to v5 means those users are lost?

Reply:
I think we can document it as a side-effect of the upgrade.

> When Grafana v5 is installed it will not inherit the user database from the old v4 installation, the admin password will be new, and user accounts set up in the v4 instance will need to be recreated in the v5 instance.

Reply:
OK, does this make it a major release?

Reply:
@durera , @whitfiea - I've updated the readme

Reply:
Nah, we're still fine as a minor